roles/rhsm-repos/defaults/main.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-openshift_required_repos: ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms', 'rhel-7-server-ose-3.6-rpms', 'rhel-7-fast-datapath-rpms']
+openshift_required_repos: ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms', 'rhel-7-server-ose-3.9-rpms', 'rhel-7-fast-datapath-rpms']
```

Comment:
I wonder that new required repository 'rhel-7-server-ansible-2.4-rpms' is needed.

Reply:
On OCP 3.9 ansible comes from the `rhel-7-server-extra-rpms` repo.

Reply:
Interesting. ansible 2.4 is also in the `rhel-7-server-extra-rpms` repo so adding `rhel-7-server-ansible-2.4-rpms` seems redundant. Nevertheless I'll update my PR to include that repo per the docs.